### ISSUE TITLE
Token interpretation inconsistency

### ISSUE THREAD

**Issue by LuisBurdallo** [jags111/efficiency-nodes-comfyui#80]
I thought it should at least get it as close as possible but after some testing it I can't really match the exact A1111 generation. There must be more to it. I am attaching a couple of pictures and maybe you can tell me if I am doing anything wrong. I've tried mean, length etc... Also all the other conversion modes. Can you help with this huge problem which is not being able to fully migrate to ComfyUI as it feels pretty inconsistent and I got some beautiful images created that I wish I could port over.

![Snippet_002](https://github.com/jags111/efficiency-nodes-comfyui/assets/33180658/f156a3e4-ac95-4fee-9639-960b55f7ffbc)

![Snippet_003](https://github.com/jags111/efficiency-nodes-comfyui/assets/33180658/bdd6b9ef-43be-4f9d-9eda-9d091157f884)

**Comment by jags111** [comment]
this issue you are referring relates to basic comfyUI sampler configuration and not in the eff scripts. so very little can be done to change same as the code in comfy works in a different way than auto1111. i have little control on same.

**Comment by LuisBurdallo** [comment]
Yes I know it is an issue by design and how ComfyUI handles weights. I just thought that the A1111 interpretation would yield the same results but I guess that is not gonna happen ever as it's developer seems to be totally against it.

**Comment by jags111** [comment]
looking into it and need a few days to resolve same
